docs/setup-cluster/checklists/_index.rst

```diff
@@ -315,11 +315,47 @@ Non-Root Containers
 ===================
 
 You can enhance security and limit potential malicious activity by running containers as non-root
-users. To find out more, visit :ref:`run-as-user` and :ref:`run-unprivileged-tasks`.
+users.
 
 .. important::
 
-   Red Hat OpenShift users should not configure non-root containers.
+   Red Hat® OpenShift® users users should not configure non-root containers.
+
+To run containers as non-root users, you'll first need to set up your non-root user:
+
+-  Choose an existing non-root user or create a new one for running containers.
+
+-  Link this user to the actual username/UID and groupname/GID. Use the following command:
+
+   .. code:: bash
+
+      det user link-with-agent-user
+         --agent-user $THE_USER --agent-uid $THE_UID --agent-group $THE_GROUP --agent-gid $THE_GID
+         $THE_DETERMINED_USER
+
+-  Start a shell as the specified user:
+
+   .. code:: bash
+
+      det -u $THE_DETERMINED_USER shell start
+
+-  In the shell, verify the username/UID and groupname/GID with ``id -a``.
+
+-  After confirming the non-root containers are operational, run normal training jobs as the
+   modified Determined user. This ensures applications function without root privileges.
+
+**Additional Steps**
+
+-  Ensure that the non-root user has the necessary permissions for required directories and files.
```

Comment:
Necessary permissions where?  Inside the container?  Outside the container?  In a shared file system?  Why is this additional when we already had a "Run your normal jobs as the nonroot user to test them" step earlier?

Reply:
good point, i removed this so that it is no longer an additional step